assets/src/edit-story/karma/elementTransform.cuj.karma.js

```diff
@@ -63,6 +63,36 @@ describe('CUJ: Creator can Transform an Element', () => {
     });
 
     describe('Action: Resize', () => {
+      it('it should not allow resizing text below 8px font size', async () => {
+        // Test that resize handle exists in edit mode.
+        const bottomRightResizeHandle = fixture.container.querySelector(
+          '.moveable-control.moveable-se'
```

Comment:
It would be nice if this was `fixture.editor.canvas.movable.resizeSE` or something like that, but it's not a blocker.

Reply:
Alright https://github.com/google/web-stories-wp/pull/3563/commits/c5ea55523776a53ac0b664c2fbe7bbc48aeb91b7